WebPortal/Pages/Project/DataProject/DataProjectPage.razor

```diff
@@ -20,7 +20,7 @@
             <CascadingValue Value=@_isDatahubAdmin Name="IsDatahubAdmin">
                 <CascadingValue Value=@_user Name="GraphUser">
                     <CascadingValue Value=@_projectId Name="ProjectId">
-                        <ProjectInfo />
+                        <ProjectInfo ProjectBudget=500 />
```

Comment:
This may just be my personal preference, but I would use a variable here and populate it with a dummy service call in OnInit, just to be clear that it's not meant to be a hardcoded value